### DISCUSSION TITLE
How to play totem of undying effect to player?

### DISCUSSION THREAD

**Discussion by skies-unbound** [Minestom/Minestom#1418] [Help] [answered]
I am trying to create a totem of undying animation with custom model data, but I see TOTEM_OF_UNDYING in particles only. It seems like there is no such way to play these animations to the player, doesn't it? I am looking for a method that can send a packet and ask the client to play a totem of undying animation.

**Comment by FluxCapacitor2** [comment, marked as answer]
You can send an entity event packet (https://wiki.vg/Protocol#Entity_Event)
with the correct ID for totem of undying: https://wiki.vg/Entity_statuses#Living_Entity

Edit: If you want other players to see the totem effect, you should use the `Entity#triggerStatus` method.
